### ISSUE TITLE
Using Corrected Trajectory

### ISSUE THREAD

**Issue by jkhall** [navisens/Android-SDK#8]
Hi. I see that you can configure "corrected_trajectory" as true when you initialize the SDK. The docs say that it will show a path. I'm wondering how to use this feature because I'm interested to see when corrections are happening.

Thanks.

**Comment by jamesgrantham** [comment]
The corrected_trajectory configuration option issues updates that can be applied to past positions. Corrections to current position will still happen with this feature turned off, but with it turned on we are able to, under some circumstances, show not just changes to where we think your are now, but also changes to where we think you have been. Keep in mind this will be useful only in a global context using the latitude and longitude provided by our SDK, and has no effect if you are using us only for the cartesian positioning. 

As an example, if you, as a user of the SDK, are outside and walking long enough for our software to determine with more certainty your position (indicated by the motiondna.getLocation().global.accurracy variable switching from LOW to HIGH), then, if corrected_trajectory is set to true, a quantity of backdated motionDna objects will be sent to the receiveMotionDna() method giving a more granular picture of where you have been back to when you first activated the SDK. This series of positions should form a much smoother path than the typical scatterplot of gps points that was previously coming through.

**Comment by jkhall** [comment]
Okay. Thanks. To be sure that I understand, with this turned to true, if accuracy increases, you will get motionDna in the receiveMotionDna callback pertaining to older portions of your path. Is there a way to recognize that this motionDna is "older", so that you could alter a path you had been drawing or whatnot?

**Comment by jamesgrantham** [comment]
Thats right! The best way to recognize when a corrected path is coming through is to check the timestamps. If the difference between the current position and the last is negative that means it is an old timestamp and that the position that just came through is correction for where the user was at that point in time.
